x/gov/test_common.go

```diff
@@ -155,6 +156,30 @@ func testProposal() Content {
 	return NewTextProposal("Test", "description")
 }
 
+type contextKey int
+
+const contextKeyBadProposalKey contextKey = iota
```

Comment:
an island `iota` doesn't really make sense (will always be `0`), something along the lines of `const contextKeyBadProposalKey = 0x00` seems cleaner (and just remove `contextKey` altogether)

Reply:
true true...I'll just make it a string.